cluster/gce/gci/configure-helper.sh

```diff
@@ -1738,6 +1758,8 @@ function prepare-etcd-manifest {
   sed -i -e "s@{{ *etcd_cluster *}}@$etcd_cluster@g" "${temp_file}"
   sed -i -e "s@{{ *liveness_probe_initial_delay *}}@${ETCD_LIVENESS_PROBE_INITIAL_DELAY_SEC:-15}@g" "${temp_file}"
   sed -i -e "s@{{ *listen_client_ip *}}@${ETCD_LISTEN_CLIENT_IP:-127.0.0.1}@g" "${temp_file}"
+  sed -i -e "s@{{ *etcd_listen_client_urls *}}@${etcd_listen_client_urls:-}@g" "${temp_file}"
+  sed -i -e "s@{{ *etcd_listen_metrics_urls *}}@${etcd_listen_metrics_urls:-}@g" "${temp_file}"
```

Comment:
Seems unneeded - there isn't etcd_listen_metrics_urls in etcd manifest.

Reply:
Removed.